frappe/public/js/frappe/form/controls/table.js

```diff
@@ -8,7 +8,7 @@ frappe.ui.form.ControlTable = class ControlTable extends frappe.ui.form.Control
 		this.grid = new Grid({
 			frm: this.frm,
 			df: this.df,
-			perm: this.perm || (this.frm && this.frm.perm) || this.df.perm,
+			_perm: this.perm || this.df.perm,
```

Comment:
I am not sure when this is ever set? 🤔 


I only found it to be using `frm.perm`.

Reply:
do we need `_perm` at all... we can figure it out inside grid. We are passing required objects (`df`, `frm`, `control`) anyway which could potentially have perm object.

let's remove it.

Reply:
Yeah makes sense. 

TODO: 
- [ ] remove duplication of perm from control and df too.